src/stream/src/common/table/test_storage_table.rs

```diff
@@ -468,3 +468,196 @@ async fn test_batch_scan_with_value_indices() {
     let res = iter.next_row().await.unwrap();
     assert!(res.is_none());
 }
+
+#[tokio::test]
+async fn test_mem_table_spill_in_batch_mode() {
```

Comment:
Same here. It is better to test the muli-version key with different spilled epochs as well.